.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+<!--- Please fill out the following template, which will help other contributors review your Pull Request. -->
+
```

Comment:
While your here, I'd suggest we drop the "Change has been approved by a ZFS on Linux member" line at the end.  We've been using the issue labels for status tracking.

Reply:
Done